Sources/Fuzzilli/Protobuf/ProtoUtils.swift

```diff
@@ -64,3 +66,56 @@ class OperationCache {
         operations.append(operation)
     }
 }
+
+public func decodeProtobufCorpus(_ buffer: Data) throws -> [Program]{
```

Comment:
Nit: can you just change the order of `encodeProtobufCorpus` and `decodeProtobufCorpus` here since encode has the nice comment at the top explaining the format =D